### PR TITLE
Add Ever Wallet Guide & Issue FT

### PR DESCRIPTION
**Pull request by babowsk1**
- https://everscale-incubator.github.io/docs/PR-297/develop/tutorial/ever-wallet/
- https://everscale-incubator.github.io/docs/PR-297/gs/firts-touch/ever-wallet/
- https://everscale-incubator.github.io/docs/PR-297/develop/tutorial/smart-digital-assets/issue-ft/

### CODE REVIEW

File: src/develop/tutorial/smart-digital-assets/issue-ft.md [hunk 1]
```diff
@@ -1,15 +1,508 @@
...
+value: locklift.utils.toNano(5),
+});
+console.log(`${name}: ${tokenRoot.address}`);
+}
+main()
+.then(() => process.exit(0))
+.catch(e => {
+console.log(e);
+process.exit(1);
+});
```
Comment (ch1seL) [inline src/develop/tutorial/smart-digital-assets/issue-ft.md:265]: formatting

File: src/develop/tutorial/smart-digital-assets/issue-ft.md [hunk 2]
```diff
@@ -1,15 +1,508 @@
...
+
+### Add Sandbox
+
+Let's add a Sandbox script to `package.json`, with which we will launch a local node.
+
+```
+ "scripts": {
+    "test": "npx locklift test --network local",
+    "start-sandbox": "docker run -d -e USER_AGREEMENT=yes --rm --name local-node -p80:80 tonlabs/local-node:0.29.1",
+    "stop-sandbox": "docker kill local-node",
```
Comment (ch1seL) [inline src/develop/tutorial/smart-digital-assets/issue-ft.md:98]: docker stop is a more friendly command

File: src/gs/firts-touch/ever-wallet.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
+---
```
Comment (ilyar) [inline src/gs/firts-touch/ever-wallet.md:1]: This page duplicates src/develop/tutorial/ever-wallet.md

File: src/develop/tutorial/smart-digital-assets/issue-ft.md (hunk 2 above)
Comment (ilyar) [reply src/develop/tutorial/smart-digital-assets/issue-ft.md:98]: use `everdev se start` and  `everdev se stop`

File: src/develop/tutorial/smart-digital-assets/issue-ft.md [hunk 4]
```diff
@@ -1,15 +1,508 @@
...
+}
+main()
+  .then(() => process.exit(0))
+  .catch(e => {
+    console.log(e);
+    process.exit(1);
+  });
+```
+
+[The TON Soidity Compiler](https://github.com/tonlabs/TON-Solidity-Compiler/blob/master/API.md#addresstransfer) allows you to specify different parameters (value,currencies, bounce or flag) of an outgoing internal message sent through an external function call.
```
Comment (ilyar) [inline src/develop/tutorial/smart-digital-assets/issue-ft.md:354]: This text is taken out of context and confuses the reader more than it explains anything. If we want to talk about the `responsible` function, then we need to make a link to the documentation for this modifier.

File: src/develop/tutorial/ever-wallet.md [hunk 5]
```diff
@@ -0,0 +1,220 @@
+---
+sidebar_position: 4
+---
+
+# Ever Wallet Guide
```
Comment (ilyar) [inline src/develop/tutorial/ever-wallet.md:5]: This page should document [this](https://github.com/EverscaleGuild/everscale-tutor-web) project https://github.com/everscale-org/docs/issues/17

File: src/develop/tutorial/smart-digital-assets/issue-ft.md [hunk 6]
```diff
@@ -1,15 +1,508 @@
 ---
 sidebar_position: 0
-[//]: # (draft: true)
 ---
 
-# Issue a Fungible Token
+# Issue a Fungible Token using Locklift
```
Comment (ilyar) [inline src/develop/tutorial/smart-digital-assets/issue-ft.md:5]: see https://github.com/everscale-org/docs/issues/257

**Review by ilyar — CHANGES_REQUESTED** [review]

File: src/develop/tutorial/smart-digital-assets/issue-ft.md (hunk 1 above)
Comment (babowsk1) [reply src/develop/tutorial/smart-digital-assets/issue-ft.md]: https://github.com/everscale-org/docs/blob/9d8035652d0f049c8184d4effc8f588e79d7123e/src/develop/tutorial/smart-digital-assets/issue-ft.md?plain=1#L235

File: src/develop/tutorial/smart-digital-assets/issue-ft.md (hunk 2 above)
Comment (babowsk1) [reply src/develop/tutorial/smart-digital-assets/issue-ft.md]: https://github.com/everscale-org/docs/blob/9d8035652d0f049c8184d4effc8f588e79d7123e/src/develop/tutorial/smart-digital-assets/issue-ft.md?plain=1#L97

File: src/gs/firts-touch/ever-wallet.md (hunk 3 above)
Comment (babowsk1) [reply src/gs/firts-touch/ever-wallet.md:1]: https://github.com/everscale-org/docs/pull/297/commits/7d234bc32ac19f7fead257ff74a8c2dce4eac33f

File: src/develop/tutorial/smart-digital-assets/issue-ft.md (hunk 4 above)
Comment (babowsk1) [reply src/develop/tutorial/smart-digital-assets/issue-ft.md]: https://github.com/everscale-org/docs/blob/9d8035652d0f049c8184d4effc8f588e79d7123e/src/develop/tutorial/smart-digital-assets/issue-ft.md?plain=1#L355

File: src/develop/tutorial/smart-digital-assets/issue-ft.md (hunk 2 above)
Comment (ch1seL) [reply src/develop/tutorial/smart-digital-assets/issue-ft.md]: so mb `npx everdev se start` like `npx locklift test --network local` command

**Review by ch1seL — APPROVED** [review]

File: src/develop/tutorial/smart-digital-assets/issue-ft.md (hunk 6 above)
Comment (ilyar) [reply src/develop/tutorial/smart-digital-assets/issue-ft.md:5]: no, you don't need to make separate pages

**Review by eternalflow — APPROVED** [review]

**Pull request merged**